.travis.yml

```diff
@@ -11,4 +11,30 @@ matrix:
 
 cache: cargo
 
-sudo: false
+sudo: true
+
+services:
+- docker
+
+install:
+- gem install fpm --no-rdoc --no-ri
+- gem install package_cloud --no-ri --no-rdoc
+
+after_success:
+- mkdir opt
+- cp target/debug/weldr opt
+# Use the same docker image as much as possible
+- docker run -v "$(pwd):/src/" cdrx/fpm-centos:7 -s dir -t deb -v 0.1.0 -n weldr -C /src  opt/weldr
```

Comment:
Should we replace the `0.1.0` version with `$TRAVIS_TAG`?

The user of docker here is fine with me. I originally thought it was going to be slow and was going to suggest doing something like `if [ -n "$TRAVIS_TAG" ]; then something when tag set; fi` but it looks quite fast.

However, now that I think about it we want to deploy the static binary version via `cargo build --target x86_64-unknown-linux-musl --release`. That is going to take a long time.